lib/retirement_swap/processor.rb

```diff
@@ -0,0 +1,30 @@
+module RetirementSwap
+  class Processor
+    class NullAlgorithm
+      def process(*args)
+        return []
+      end
+    end
+
+    ALGORITHMS = {
+      'swap' => RetirementSwap::SwapAlgorithm
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.